src/core/qgsmapsettings.cpp

```diff
@@ -682,6 +682,21 @@ void QgsMapSettings::setLabelBoundaryGeometry( const QgsGeometry &boundary )
   mLabelBoundaryGeometry = boundary;
 }
 
+void QgsMapSettings::addClippingRegion( const QgsMapClippingRegion &region )
```

Comment:
Background question: have we ever discussed whether and when we should use add vs append? I have absolutely no strong feeling either way, just wondering.

Reply:
add for "add to an unordered set", append for "append to end of an ordered list"?

Reply:
@m-kuhn -- I like that. I'll leave this one as "add", because even though value is internally stored in a list, the order isn't important and we may change that in future.